src/main/groovy/org/labkey/gradle/task/ModuleDistribution.groovy

```diff
@@ -22,36 +22,32 @@ import org.gradle.api.Project
 import org.gradle.api.file.CopySpec
 import org.gradle.api.file.DuplicatesStrategy
 import org.gradle.api.file.FileTree
+import org.gradle.api.provider.Property
 import org.gradle.api.tasks.*
 import org.labkey.gradle.plugin.ApplyLicenses
 import org.labkey.gradle.plugin.extension.DistributionExtension
 import org.labkey.gradle.plugin.extension.LabKeyExtension
 import org.labkey.gradle.plugin.extension.StagingExtension
 import org.labkey.gradle.util.BuildUtils
-import org.labkey.gradle.util.PropertiesUtils
 
 import java.nio.file.Files
 
 class ModuleDistribution extends DefaultTask
 {
-    @Optional @Input
-    Boolean includeZipArchive = false
-    @Optional @Input
-    Boolean includeTarGZArchive = false
+    // TODO: Remove this unused property
     @Optional @Input
     String embeddedArchiveType = null
-    @Optional @Input
-    Boolean makeDistribution = true // set to false for just an archive of modules
+
     @Optional @Input
     String extraFileIdentifier = ""
     @Optional @Input
     String versionPrefix = null
     @Optional @Input
```

Comment:
I think the @Optional can/should be removed here. We need some kind of name.

Reply:
Interesting... this was marked `@Optional` before my changes. What about keeping it optional and defaulting to `project.name` a la `subDirName`?

Reply:
I thought I was pointing to the `@Optional` for `subDirName` with this comment.   Did I misread?

Reply:
Sorry, I was unclear which `@Optional` was being referenced. As for `subDirName`, the code was clearly attempting to make it optional; it was annotated as such and defaulting to `project.name.` Only problem was `getArtifactId()` hadn't gotten the memo. It now correctly defaults to `project.name`, which was my goal and what we discussed over email. But maybe I'm misunderstanding your suggestion or what `@Optional` means...

Reply:
Oh, my bad. I think we do want to leave it so Gradle/Groovy knows the task doesn't require a value to be provided.